### PR TITLE
Feature/419 v1 release

### PR DESCRIPTION
**Pull request by LesterThomas**
This is the release of the canvas that includes the `v1` of the component specification. See #419

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,15 +23,35 @@ The Reference Implementation of the ODA Canvas will be used for ODA Component ce
...
 The design (including ongoing work) of the ODA Canvas is documented in [ODA Canvas Design](Canvas-design.md).
 
 
 ## Release notes
 
+
 | Version    | Release notes                         |
 |:----------:|---------------------------------------|
+| 1.2.0      | Release of `v1` component specification and general availability of ODA Canvas |
+| 1.1.7      | Updated to `v1beta4` component specification |
```
Comment (brian-burton) [inline README.md:50]: v1.1.8 from #417 is missing.

File: charts/canvas-api-gateway/combined-api-gateway-chart/Chart.lock
Comment (brian-burton) [inline charts/canvas-api-gateway/combined-api-gateway-chart/Chart.lock:1]: Should chart.lock be gitignored?

File: charts/canvas-oda/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # canvas-oda
 
-![Version: 1.1.8-lt4](https://img.shields.io/badge/Version-1.1.8--lt4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1beta4](https://img.shields.io/badge/AppVersion-v1beta4-informational?style=flat-square)
+![Version: 1.1.8-lt4](https://img.shields.io/badge/Version-1.1.8--lt4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1](https://img.shields.io/badge/AppVersion-v1-informational?style=flat-square)
```
Comment (brian-burton) [inline charts/canvas-oda/README.md:3]: Should be v1.2.0 here?

Comment (LesterThomas) [reply charts/canvas-oda/README.md]: These are auto-generated README files generated by `helm-docs` tool. I've regenerated all the README's

File: charts/canvas-api-gateway/combined-api-gateway-chart/Chart.lock
Comment (LesterThomas) [reply charts/canvas-api-gateway/combined-api-gateway-chart/Chart.lock:1]: Yes - I think so!

Comment (LesterThomas) [reply charts/canvas-api-gateway/combined-api-gateway-chart/Chart.lock:1]: I think the whole `canvas-api-gateway` folder may be 'dead wood' - @Ravi has changed how the different API Gateways are installed.

File: README.md (hunk 1 above)
Comment (LesterThomas) [reply README.md]: I've added to release notes

**Review by brian-burton — COMMENTED** [review]
Looks good tm me overall, but with comments to be resolved before approval.

File: charts/oda-crds/templates/oda-component-crd.yaml [hunk 3]
```diff
@@ -15,38 +15,53 @@ spec:
...
   versions:
-  - name: "v1beta1"
+  - name: "v1beta2"
     # Each version can be enabled/disabled by Served flag.
     served: true
     # One and only one version must be marked as the storage version.
     storage: false
     deprecated: true
-    deprecationWarning: "oda.tmforum.org/v1beta1 Component is deprecated. Please upgrade to oda.tmforum.org/v1beta3 or newer"
+    deprecationWarning: "oda.tmforum.org/v1beta2 Component is deprecated. Please upgrade to oda.tmforum.org/v1 or newer"
```
[the remark this answers is not in the document]
Comment (LesterThomas) [reply charts/oda-crds/templates/oda-component-crd.yaml:32]: I intentionally put the recommendation to upgrade to v1 (and the or newer refers to future releases).

File: charts/oda-crds/templates/oda-publishednotification-crd.yaml [hunk 4]
```diff
@@ -10,12 +10,13 @@ spec:
...
   versions:
-    - name: "v1beta1"
+    - name: "v1beta2"
       # Each version can be enabled/disabled by Served flag.
       served: true
       # One and only one version must be marked as the storage version.
       storage: false
-      deprecated: false
+      deprecated: true
+      deprecationWarning: "oda.tmforum.org/v1beta2 PublishedNotification is deprecated. Please upgrade to oda.tmforum.org/v1 or newer"
```
[the remark this answers is not in the document]
Comment (LesterThomas) [reply charts/oda-crds/templates/oda-publishednotification-crd.yaml:19]: I intentionally put the recommendation to upgrade to v1 (and the or newer refers to future releases).

File: charts/canvas-api-gateway/combined-api-gateway-chart/Chart.lock
Comment (LesterThomas) [reply charts/canvas-api-gateway/combined-api-gateway-chart/Chart.lock:1]: I've removed `canvas-api-gateway`

**Pull request merged**